From 30905a0238e986dd1775e6d938605c5f6afdf858 Mon Sep 17 00:00:00 2001
From: Peleg Vigodny <peleg.vigodny@tandemg.com>
Date: Thu, 07 May 2020 10:38:35 +0300
Subject: [PATCH] [SW-10967] gaudi: linux: pca953x: enable driver

Enable PCA953x gpio expander driver in Gaudi's Linux.

This change is added as preparation for the HL200 QSFP-DD driver.
QSFP-DD modules are controlled by TCA6416 I/O expander, and we can use
the PCA953x driver to control this device.

GPIO sysfs support is also added to the configuration in order to allow
us to manually control these modules from userspace.

Change-Id: Ia31ca37780e9bb23b75f2d3e7dfb02e504b9749c
Signed-off-by: Peleg Vigodny <peleg.vigodny@tandemg.com>
---

diff --git a/configs/gaudi_defconfig b/configs/gaudi_defconfig
index 2cb3261..759a1a8 100644
--- a/configs/gaudi_defconfig
+++ b/configs/gaudi_defconfig
@@ -60,3 +60,6 @@
 BR2_PACKAGE_PREBOOT=y
 BR2_PACKAGE_BTL=y
 BR2_PACKAGE_EEPROM_UTIL=y
+CONFIG_GPIOLIB=y
+CONFIG_GPIO_SYSFS=y
+CONFIG_GPIO_PCA953X=y
